Loop/Managers/ServicesManager.swift

```diff
@@ -201,6 +205,40 @@ extension ServicesManager: ServiceDelegate {
         log.default("Service with identifier '%{public}@' deleted", service.serviceIdentifier)
         removeActiveService(service)
     }
+    
```

Comment:
The `ServiceDelegate` protocol conforms now to the `RemoteCommandHandler` and these methods are required.